config/pipelines.yml

```diff
@@ -35,6 +35,15 @@
 #   # before dispatching an undersized batch to filters+outputs
 #   pipeline.batch.delay: 50
 #
+#   Set the pipeline event ordering. Options are "auto" (the default), "true" # #   or "false".
+#   "auto" automatically enables ordering if the 'pipeline.workers' setting
+#   is also set to '1', and disables otherwise.
+#   "true" enforces ordering on the pipeline and prevent logstash from starting
+#   if there are multiple workers.
```

Comment:
Slightly different behaviour in a multi-pipeline scenario - any pipelines with `pipeline.ordered: true` and more than 1 `pipeline.workers` will fail to start, but other pipelines will start as expected.

```suggestion
#   "true" enforces ordering on a pipeline and prevents logstash from starting
#   a pipeline with multiple workers allocated.
```